File: programacion/java/1_unidad/000ejercicos_examen/ComprobarEdad.java
import java.util.Scanner;
public class ComprobarEdad {
    public static void main(String[] args) {
        Scanner teclado = new Scanner(System.in);
        System.out.print("Introduce tu edad: ");
        int edad = teclado.nextInt();

        if (edad < 18 && edad >= 0) {
            System.out.println("Eres menor de edad.");
        } else if (edad >= 18 && edad < 66) {
            System.out.println("Eres mayor de edad en edad de trabajar.");
        } else if (edad >= 66) {
            System.out.println("Eres jubilado.");
        } else {
            System.out.println("Edad no válida.");
        }
        teclado.close();
    } 
}

File: programacion/python/unidad_1/000ejercicios_examen/ComprobarEdad.py
edad = int(input("Introduce tu edad: "))
if edad >= 18:
    print("Eres mayor de edad.")
elif edad > 66:
    print("Eres un jubilado.")
elif edad < 0:
    print("Edad no válida.")
else:
    print("Eres menor de edad.")